src/cow-react/modules/limitOrders/pure/OrderStatusBox/index.tsx

```diff
@@ -48,3 +55,31 @@ export const OrderStatusBox = styled.div<{ status: OrderStatus; cancelling: bool
     border-radius: ${({ withWarning }) => (withWarning ? '9px 0 0 9px' : '9px')};
   }
 `
+
+export type OrderStatusBoxProps = { order: ParsedOrder; widthAuto?: boolean }
+
+export function OrderStatusBox({ order, widthAuto }: OrderStatusBoxProps) {
+  return (
+    <Wrapper
+      style={widthAuto ? { width: 'auto' } : {}}
```

Comment:
it was already there, but in principle, we should style things using props of styled components, not using the raw style

Reply:
Good point, will do